src/MissionEditor/SimpleItemEditor.qml

```diff
@@ -119,6 +119,13 @@ Rectangle {
                 onClicked:  editorRoot.moveHomeToMapCenter()
                 anchors.horizontalCenter: parent.horizontalCenter
             }
+
+            QGCButton {
```

Comment:
After changing latitude and longitude home position, the map center also stays in the old view. But I want change view to the new home position. So I add this Button for speeding up view changing.